src/python/test/test_persistence_graphical_tools.py

```diff
@@ -98,8 +98,12 @@ def test_limit_to_max_intervals():
 def _limit_plot_persistence(function):
     pplot = function(persistence=[()])
     assert issubclass(type(pplot), plt.axes.SubplotBase)
+    pplot = function(persistence=[()], legend=True)
```

Comment:
`[()]` looks like an invalid value for `persistence` according to the doc. The empty case would be `[]`. Am I missing something?

Reply:
You are completely right. I modified code and test on [393216c](https://github.com/GUDHI/gudhi-devel/pull/604/commits/393216cc99af79526b1bb90ae4da21116884bcbd)